src/middleware.ts

```diff
@@ -1,6 +1,16 @@
 import { NextRequest, NextResponse } from "next/server";
 
 export function middleware(req: NextRequest) {
+    // poroto.appの場合はkomichi.appにリダイレクト
+    // TODO: 2024年10月以降に削除
+    const { hostname } = req.nextUrl;
+    if (hostname === "poroto.app") {
+        const url = new URL(
+            `${process.env.APP_PROTOCOL}://${process.env.APP_HOST}/`
+        );
+        return NextResponse.redirect(url, 301);
+    }
```

Comment:
ここを以下のように変更すると`localhost:3000`にアクセスしたときに`komichi.app`にリダイレクトされることを確認できます。
```suggestion
    const { hostname } = req.nextUrl;
    if (hostname === "localhost") {
        const url = new URL(
            `https://komichi.app/`
        );
        return NextResponse.redirect(url, 301);
    }
```